Core/Loaders/ChestLootSystemLoader.cs

```diff
@@ -0,0 +1,26 @@
+﻿using StarlightRiver.Content.Alchemy;
+using StarlightRiver.Content.WorldGeneration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StarlightRiver.Core.Loaders
+{
+    public class ChestLootSystemLoader : IOrderedLoadable
```

Comment:
What is the purpose of this? If we're only loading a single system, why not simply make that system loadable?